### ISSUE TITLE
Can't send any amount of coins

### ISSUE THREAD

**Issue by devzorg** [iron-fish/ironfish#1348]
### What happened?

Can't send any amount of IronFish coins to 5cfbcbf6e6f84e2bab6c17309f39908d7cdb8ae5ad4ab600bbb9101403a1edc2d03f71754764e8c66afc08
```
#> ironfish accounts:balance
Account - rad1k (ab56fd7)

The balance is: $IRON 72.10000467 ($ORE 7,210,000,467)
Amount available to spend: $IRON 52.10000244 ($ORE 5,210,000,244)
```

### Debug output

```shell
Database in use, skipping output that requires database.
Stop the node and run the debug command again to show full output.

Iron Fish version       0.1.32 @ 1abccc9
Iron Fish library       0.0.10 @ 1abccc9
Operating system        Linux x64
CPU model(s)            AMD Ryzen 9 5950X 16-Core Processor
CPU threads             32
RAM total               62.78 GiB
Node version            v16.13.0
ironfish in PATH        true
Telemetry enabled       true
```

### Relevant log output

```shell
the transaction: [████████████████████████████████████████] 100% | ETA: 0s
An error occurred while sending the transaction.
 ›   Error: Request failed (400) insufficient-balance: Your balance is too low. Add funds to your account first
```

### Graffiti

rad1k

**Comment by roadswithoutend** [comment]
I have the same issue
The balance is: $IRON 68.39998985 ($ORE 6,839,998,985)
Amount available to spend: $IRON 28.39998832 ($ORE 2,839,998,832)
Done in 27.53s.

You are about to send $IRON 0.10000000 ($ORE 10,000,000) plus a transaction fee of $IRON 0.00000001 ($ORE 1) to the Iron Fish deposit account.
Your remaining balance after this transaction will be $IRON 28.29998831 ($ORE 2,829,998,831).
The memo will contain the graffiti "roadswithoutend".

* This action is NOT reversible *
      
Do you confirm (Y/N)?: y
Creating the transaction: [████████████████████████████████████████] 100% | ETA: 0s
An error occurred while sending the transaction.
 ›   Error: Request failed (400) error: Insufficient funds
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

**Comment by lwisne** [comment]
We believe this is fixed now. Comment here if you're still experiencing this issue!
